R/summary_functions.R

```diff
@@ -14,13 +15,18 @@ percent_coverage <- function(coverage_result){
 
 #' Provide locations of zero coverage
 #'
-#' When examining the test coverage of a package, it is useful to know if there are 
+#' When examining the test coverage of a package, it is useful to know if there are
 #' any locations where there is \bold{0} test coverage.
 #'
-#' @param coverage_result a coverage object returned from \code{\link{package_coverage}}
+#' @param coverage_result a coverage object returned from
+#' 	\code{\link{package_coverage}}, or its data frame conversion
 #' @export
-zero_coverage <- function(coverage_result){
-  coverage_data <- as.data.frame(coverage_result)
+zero_coverage <- function(coverage_result) {
+  coverage_data <- if (is.data.frame(coverage_result)) {
```

Comment:
This test is unnecessary, if an object is already a `data.frame` `as.data.frame` will just return the object without doing anything if `x` is already a `data.frame` (Assuming the `row.names` are not `NULL`).